js/blocks/controls/fetch-input.js

```diff
@@ -141,8 +136,11 @@ class FetchInput extends Component {
 		} else {
 			this.inputRef.current.setCustomValidity( '' );
 		}
-	}
 
+		this.inputRef.current.className = classNames( 'bl-fetch__input', {
+			'text-control__error': ! isValid
```

Comment:
Also add a red border if the input is invalid (which is what we do with other fields).

Reply:
Nice, that looks good.